test/box-luatest/gh_4691_data/fill.lua

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env tarantool
+
+local fio = require('fio')
+
+-- {{{ Helpers to determine the script directory
+
+local function sourcefile()
+    local info = debug.getinfo(3, 'S')
+    local source = info and info.source
+    return source and source:startswith('@') and source:sub(2) or nil
+end
+
+local function sourcedir()
+    local source = sourcefile()
+    return fio.abspath(source and source:match('(.*)/') or '.')
+end
```

Comment:
Nit: this seems excess. Below anyway you do
```Lua
local snaps = fio.glob('*.snap')
assert(#snaps == 1)
```
which assumes the work dir == WAL dir. Hence you could also just drop `workdir` arg and this code.

Reply:
It seems, I missed your point. I want to generate the snapshot in the directory, where the script resides. I don't want to manually copy the snapshot from a current directory.

It is opinionated, but I like that the script doing all the needed work itself. Not some random part of the work, while leaving some other random part of the work to a human. That's disgusting. Let a machine doing that.

Reply:
I somewhy thought that `workdir` won't change workdir of the whole process, all is fine.